topics/variant-analysis/tutorials/tb-variant-analysis/tutorial.md

```diff
@@ -308,13 +309,18 @@ We could go through all of the variants in the VCF files and read them out of a
 
 > ### {% icon hands_on %} Hands-on: Run JBrowse
 >
-> 1. {% tool [JBrowse](toolshed.g2.bx.psu.edu/repos/iuc/jbrowse/jbrowse/1.16.8+galaxy1) %} {% icon tool %} with the following parameters
+> 1. Use {% tool [seqret](tool_id=toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: seqret84/5.0.0) %} {% icon tool %} to convert the Genbank format reference (`Mycobacterium_tuberculosis_ancestral_reference.gbk`) to FASTA format. Use the following parameters:
```

Comment:
@hexylena a tool ID with a space in it! Will that work or should the space be encoded or sth?

Reply:
Eesh. 

1. remove `tool_id=`, that's unnecessary
2. It *should* work, but safer to write a `+`.
3. this never should have been allowed :| 

[I've tested it just now](https://usegalaxy.eu/?tool_id=toolshed.g2.bx.psu.edu/repos/devteam/emboss_5/EMBOSS: seqret84/5.0.0) and it seems to auto-replace with a `+`

Reply:
lol, markdown can't handle that space. Unsurprising.

Reply:
So the space gets translated into `+` as you've said, but the `+` version fails to locate the tool then.
`%20` seems to be ok for markdown, but reaches Galaxy as a space and triggers `+` replacement just the same.
I'd love to test `%2520` but I'm not sure how to test that.

Reply:
Ah I was looking at the wrong PR, https://github.com/galaxyproject/galaxy/pull/13347 yeah, %20 is the right answer.

I'll add linting for it on our end.

Reply:
> Ah I was looking at the wrong PR, https://github.com/galaxyproject/galaxy/pull/13347 yeah, %20 is the right answer.

Thanks, should work from looking at the generated html, yes. Guess was my editor that got me confused even further :)